src/cloudant/database.py

```diff
@@ -251,57 +251,72 @@ def get_view_result(self, ddoc_id, view_name, **kwargs):
 
     def get_view_raw_result(self, ddoc_id, view_name, **kwargs):
         """
-        _get_view_raw_result_
-
-        Returns the raw response JSON content for the view query
-        based on the design document, view name and parameters
-        depicted as the kwargs.
-
-        For example to retrieve the full resulting set of raw data
-        from a design document view do:
-
-        db.get_view_raw_result('_design/ddoc_id_001', 'view_001')
-
-        or to provide parameters to the view query and return a
-        resulting set of raw data do something like:
-
-        db.get_view_raw_result('_design/ddoc_id_001', 'view_001',
-            include_docs=True, reduce=False)
-
-        :param ddoc_id: Design document id
-        :param view_name: Name of the view
-        :param **kwargs: Parameters to index the query results by
-            Valid parameters include:
-
-            descending bool
-            endkey string or array
-            endkey_docid  string
-            group bool
-            group_level ??
-            include_docs bool
-            inclusive_end  bool
-            key string
-            keys list
-            limit   int
-            reduce  boolean
-            skip    int
-            stale   enum(ok, update_after)
-            startkey  string or array
-            startkey_docid  string
+        Retrieves the raw JSON content based on the design document
+        and view name.  Unlike
+        :func:`~cloudant.database.CouchDatabase.get_view_result` the use
+        of ``skip`` and ``limit`` as kwargs is valid and
+        actually necessary in order to manage paging and iteration.
+        If you would like paging and iteration handled automatically for you
+        then use the
+        :func:`~cloudant.database.CouchDatabase.get_view_result`
+        method instead.
+
+        For example to retrieve the raw JSON response content based on a
+        design document view do:
+
+        .. code-block:: python
+
+            db.get_view_raw_result('_design/ddoc_id_001', 'view_001')
+
+        But to retrieve the raw JSON response content based on a set of
+        parameters on the same design document view do something like:
+
+        .. code-block:: python
+
+            db.get_view_raw_result('_design/ddoc_id_001', 'view_001',
+                include_docs=True, skip=100, limit=100, reduce=False)
+
+        :param str ddoc_id: Design document id used to get result.
+        :param str view_name: Name of the view used to get result.
+        :param bool descending: Return documents in descending key order.
+        :param endkey: Stop returning records at this specified key.  Can be
+            either a ``str`` or ``list``.
```

Comment:
That makes sense.  I added the complex key wording to get_view_result, get_view_raw_result and, all_docs method docstrings.  Also I replaced "JSON arrays" with "lists" in the group_level parameter description since we are passing Python parameters and not their JSON equivalent.  This is done in afb46374a18b3ac03749ba1e8638c82679b07985.